test/test_cases/run_2d_moist_bubble.jl

```diff
@@ -0,0 +1,148 @@
+if !haskey(ENV, "BUILDKITE")
+    import Pkg
+    Pkg.develop(Pkg.PackageSpec(; path = dirname(dirname(@__DIR__))))
+end
+using Test
+
+using OrdinaryDiffEq: SSPRK33, CallbackSet
+using ClimaCorePlots, Plots
+using UnPack
+
+
+using CLIMAParameters
+using ClimaAtmos.Utils.InitialConditions: init_2d_moist_bubble
+using ClimaAtmos.Domains
+using ClimaAtmos.BoundaryConditions
+using ClimaAtmos.Models
+using ClimaAtmos.Models.Nonhydrostatic2DModels
+using ClimaAtmos.Callbacks
+using ClimaAtmos.Simulations
+
+
+# Set up parameters
+struct Bubble2DParameters <: CLIMAParameters.AbstractEarthParameterSet end
+
+function run_2d_moist_bubble(
+    ::Type{FT};
+    stepper = SSPRK33(),
+    nelements = (20, 80),
+    npolynomial = 4,
+    dt = 0.02,
+    callbacks = (),
+    test_mode = :regression,
```

Comment:
Can we run this in validation mode so that we can see the results?

Reply:
We did. It crashes somewhere halfway through. I will try to debug next week. But first I have to figure out how to actually save the output of those simulations. :/